include/grpcpp/impl/codegen/sync.h

```diff
@@ -143,13 +143,6 @@ class CondVar {
 
 #endif  // GPR_ABSEIL_SYNC
 
-template <typename Predicate>
-static void WaitUntil(CondVar* cv, Mutex* mu, Predicate pred) {
```

Comment:
This is public API (I think it should not be, but it is)
We probably need at least a gRFC for this.

Reply:
I can restore this function, if you think that's preferable. Should I mark it deprecated?